src/js/pinny.js

```diff
@@ -153,6 +158,9 @@
             this.$pinny.shade('destroy');
             this.$pinny.remove();
 
+            SyntheticResize.destroy();
+            $(window).off(events.resize);
```

Comment:
So we're basically doing both of these things here? Do we want to monitor for resize AND do the synthetic resize polling? If so, this should be behind a separate function that hides this stuff.